src/video_emotion/emotionTagger/face_emotion_extraction.py

```diff
@@ -2,22 +2,61 @@
 import numpy
 from keras.models import load_model
 from keras import backend as K
+from typing import Dict
 
 from utilities.filehandler.handle_path import get_absolute_path
 
 
-def get_labels():
+def get_labels() -> Dict[int, str]:
+
+    """Get the labels for each argument in the emotion array
+
+        Returns
+        -------
+        string
+            Returns the labels for each emotion as a dictionary
+        """
+
     return {0: 'angry', 1: 'disgust', 2: 'fear',
             3: 'happy', 4: 'sad', 5: 'surprise',
             6: 'neutral'}
 
 
-def preprocess_input(x):
+def preprocess_input(x: float) -> float:
+    """Preprocesses input
+
+    Parameters
+    ----------
+    float
+        The float to be proccessed
+
+
+            Returns
+            -------
+            float
+                Returns the input, but preprocessed
+            """
+
     x = x.astype('float32')
     return ((x / 255.0) - 0.5) * 2.0
 
 
 def classify_faces(faces):
+
```

Comment:
Remove this line :) And add return type and type of the *faces* parameter if possible :)